### PR TITLE
DBKUBE-323 Don't tell users to create a custom package in the upgrade docs

### PR DESCRIPTION
**Pull request by apawelczyk-atlassian**
Currently, we have a lot of unnecessary duplication in the docs because there are more similarities than differences in the upgrade process for various products. We're thinking how to restructure the upgrade docs to have one guide with separate chapters for Bitbucket, Confluence and Jira explaining different ways of handling ZDU.

### CODE REVIEW

File: docs/product_upgrades/JIRA_UPGRADE.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+2. Go to **Administration > Applications > Jira upgrades** and click **Put Jira into upgrade mode**.
+
+  ![upgrade-mode](../images/jira-upgrade-1.png)
+
+3. Run helm upgrade command with the tag from previous steps (`<target-tag>`) and the desired number of nodes after
+   upgrade (`<nodes>`):
+
+ ```shell script
+ $ helm upgrade --wait <release name> --reuse-values --set replicaCount=<nodes> --set image.tag=<target-tag> target/helm/jira-<chart-version>.tgz
```
Comment (apawelczyk-atlassian) [inline docs/product_upgrades/JIRA_UPGRADE.md:19]: The ".tgz" file can be replaced with "atlassian-data-center/jira".

**Review by nanux — APPROVED** [review]

**Comment by nanux** [comment]
There is a simple conflict in the path as @tan-ro moved the documentation to a different folder.

File: docs/product_upgrades/JIRA_UPGRADE.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# Jira Rolling Upgrade
+For the sake of example, let's say we have Jira in version `8.13.0` deployed to our Kubernetes cluster that we want to
```
Comment (tan-ro) [inline docs/product_upgrades/JIRA_UPGRADE.md:2]: I would change "For the sake of example, let's say" to "Let's say"

File: docs/product_upgrades/JIRA_UPGRADE.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+upgrade it to `8.13.1` which we'll call the *target version*. You can substitute the target version for the one you
```
Comment (tan-ro) [inline docs/product_upgrades/JIRA_UPGRADE.md:3]: I would change to:
Let's say we have Jira version `8.13.0` deployed to our Kubernetes cluster, and we want to upgrade it to version `8.13.1`, which we'll call the *target version*.

File: docs/product_upgrades/JIRA_UPGRADE.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+need, as long as it's newer than the current one.
+
+## 1. Find tag of the target image
+
+Go to [atlassian/jira-software](https://hub.docker.com/r/atlassian/jira-software/tags)
+Docker Hub page to pick a tag that matches your target version.
+
+In the example, we're running Jira using the `8.13.0-jdk11` tag, and we'll be upgrading to `8.13.1-jdk11` - our *target*.
```
Comment (tan-ro) [inline docs/product_upgrades/JIRA_UPGRADE.md:11]: In the example we're

File: docs/product_upgrades/JIRA_UPGRADE.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+## 2. Put Jira into upgrade mode
+
+Go to **Administration > Applications > Jira upgrades** and click **Put Jira into upgrade mode**.
+
+  ![upgrade-mode](../images/jira-upgrade-1.png)
+
+## 3. Run the upgrade using Helm
+
+Run Helm *upgrade* command with your release name (`<release-name>`) and the target image from a previous step
+(`<target-tag>`). For more details, consult the official Helm documentation.
```
Comment (tan-ro) [inline docs/product_upgrades/JIRA_UPGRADE.md:22]: consult the [Helm documentation](https://helm.sh/docs/).

**Review by tan-ro — APPROVED** [review]

File: docs/product_upgrades/JIRA_UPGRADE.md (hunk 3 above)
Comment (apawelczyk-atlassian) [reply docs/product_upgrades/JIRA_UPGRADE.md:3]: I didn't like that sentence. Thanks; it's much better now.

**Pull request merged**